public/scripts/extensions/caption/settings.html

```diff
@@ -54,36 +54,36 @@
+                        <option data-type="ollama" value="ollama_current" data-i18n="[Currently selected]">[Currently selected]</option>
                         <option data-type="ollama" value="bakllava:latest">bakllava:latest</option>
                         <option data-type="ollama" value="llava:latest">llava:latest</option>
-                        <option data-type="llamacpp" value="llamacpp_current">[Currently loaded]</option>
-                        <option data-type="ooba" value="ooba_current">[Currently loaded]</option>
-                        <option data-type="koboldcpp" value="koboldcpp_current">[Currently loaded]</option>
-                        <option data-type="custom" value="custom_current">[Currently selected]</option>
+                        <option data-type="llamacpp" value="llamacpp_current" data-i18n="[Currently loaded]">[Currently loaded]</option>
+                        <option data-type="ooba" value="ooba_current" data-i18n="[Currently loaded]">[Currently loaded]</option>
+                        <option data-type="koboldcpp" value="koboldcpp_current" data-i18n="[Currently loaded]">[Currently loaded]</option>
+                        <option data-type="custom" value="custom_current" data-i18n="[Currently selected]">[Currently selected]</option>
                     </select>
                 </div>
                 <label data-type="openai,anthropic,google" class="checkbox_label flexBasis100p" for="caption_allow_reverse_proxy" title="Allow using reverse proxy if defined and valid.">
                     <input id="caption_allow_reverse_proxy" type="checkbox" class="checkbox">
-                    Allow reverse proxy
+                    <div data-i18n="Allow Reverse Proxy">Allow Reverse Proxy</div>
                 </label>
                 <div class="flexBasis100p m-b-1">
-                    <small><b>Hint:</b> Set your API keys and endpoints in the 'API Connections' tab first.</small>
+                    <small data-i18n="&lt;b&gt;Hint:&lt;/b&gt; Set your API keys and endpoints in the &#39;API Connections&#39; tab first."><b>Hint:</b> Set your API keys and endpoints in the 'API Connections' tab first.</small>
                 </div>
             </div>
             <div id="caption_prompt_block">
-                <label for="caption_prompt">Caption Prompt</label>
+                <label for="caption_prompt" data-i18n="Caption Prompt">Caption Prompt</label>
                 <textarea id="caption_prompt" class="text_pole" rows="1" placeholder="&lt; Use default &gt;">${PROMPT_DEFAULT}</textarea>
                 <label class="checkbox_label margin-bot-10px" for="caption_prompt_ask" title="Ask for a custom prompt every time an image is captioned.">
                     <input id="caption_prompt_ask" type="checkbox" class="checkbox">
-                    Ask every time
+                    <div data-i18n="Ask every time">Ask every time</div>
```

Comment:
Replace div with span

Reply:
fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/12c5466f08555ea88e350ba5aff0572878e44ba7